test/utils.js

```diff
@@ -165,6 +165,27 @@ const fixtures = {
 		return Promise.all(Array.from({
 			 length: numberOfTags 
 		}).map(() =>  fixtures.insertTag()))
+	},
+	async insertPostComment(overrides = {}){
+		const commentData = {
+			comment : faker.random.words()
+		}
+		const newData = {...commentData, ...overrides}
+		const result = await db.query(
+			`SELECT * FROM posts 
+		WHERE id = $1
+		AND type = $2 `,
+		 [newData.id , newData.type ])
+		const post = result.rows[0]
```

Comment:
Cool but we won't be needing this part since we only want to insert a comment and not necessarily get a post. 

this seems like the logic in the actual service and that was done because of the API response we needed to return. Here we only need a comment inserted